### PR TITLE
refactor(portal): Return single relay with multiple addresses

### PR DESCRIPTION
**Pull request by AndrewDryga**

### CODE REVIEW

File: elixir/apps/api/lib/api/client/views/relay.ex [hunk 1]
```diff
@@ -5,31 +5,22 @@ defmodule API.Client.Views.Relay do
...
   # so there is no need to send both of them along with each other.
 
-  defp maybe_render(%Relays.Relay{} = relay, _expires_at, address, :stun) do
+  def render(%Relays.Relay{} = relay, _expires_at, :stun) do
     [
       %{
         type: :stun,
-        addr: "#{format_address(address)}:#{relay.port}"
+        ipv4_addr: if(relay.ipv4, do: "#{format_address(relay.ipv4)}:#{relay.port}"),
+        ipv6_addr: if(relay.ipv6, do: "#{format_address(relay.ipv6)}:#{relay.port}")
```
Comment (AndrewDryga) [inline elixir/apps/api/lib/api/client/views/relay.ex]: This is the change that you will see: `ipv4_addr` and `ipv6_addr` for each relay, either of those can be `null`.

**Comment by thomaseizinger** [comment]
Thanks! I'd hold off on changing this for now until we revisit, how want to do fallback, redundancy etc.

**Comment by jamilbk** [comment]
Stale issue, converting to draft

**Comment by jamilbk** [comment]
Can this be closed?

**Comment by thomaseizinger** [comment]
> Can this be closed?

We can close it for now. Eventually, we should make a (backwards-compatible) change to the message to remove some cruft in the clients.

**Comment by jamilbk** [comment]
Closing due to inactivity

**Pull request closed without merging**